starwars/src/components/StarwarsCard.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import { Card, CardBody, CardTitle, CardText, Col } from "reactstrap";
+
+const StarwarsCard = props => {
```

Comment:
great work here!! awesome use of reactstrap and good work breaking your code into components that make sense. Note that you can make your code more DRY by passing the whole object instead of each individual key and destructuring, like this:
`const { name, birth_year, skinColor, hairColor, homeWorld, gender } = props.person`